test/masq.test.js

```diff
@@ -2,9 +2,9 @@ const signalserver = require('signalhubws/server')
 const signalhub = require('signalhubws')
 const swarm = require('webrtc-swarm')
 const wrtc = require('wrtc')
-window.crypto = require('@trust/webcrypto')
 const common = require('masq-common')
 const ERRORS = common.errors.ERRORS
+window.crypto = require('@trust/webcrypto')
```

Comment:
why move this line ?

Reply:
in order to gather imports types together. `const` together, window.[object], then relative imports below

Reply:
I would have gathered them based on the type of modules that are imported instead of the types of imports:
- third party modules
- our own module
- internal imports (relative path imports)

Reply:
well yeah I wanted to swap position between `const common...` and `window.crypto...` instead. Did not paid attention to `const ERROR = common.errors.ERRORS` not being an module import. Will fix

Reply:
I'm unable to make it work. I will use your solution @sujeebant thanks

Reply:
I find It's too verbose to surround with try catch throught, if you find another solution ;)